services/mailservice/mail.go

```diff
@@ -345,20 +357,6 @@ func SendMail(c smtpClient, mail mailData, fileBackend filesstore.FileBackend, d
 		m.EmbedReader(name, reader)
 	}
 
-	for _, fileInfo := range mail.attachments {
```

Comment:
So are we dropping support for attachments?

Reply:
We are dropping "not exposed/not used" file attachments in the emails (except for the embedded files) that are supported and handled through other way. If we want to add back attachments in the future, I would recommend to add them without depending explicitly from the filestoreage package by itself (the coupling there wasn't needed event if we wanted to attach files).

Reply:
Sounds good :+1: We can always add a method in the future that takes files/readers.